#!/usr/bin/env python

PACKAGE = "mur_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_z", double_t,   0, "Proportional parameter Z",       1., 0, 100)
gen.add("d_z", double_t,   0, "Derivative parameter Z",         1., 0, 100)

gen.add("pos_z", double_t,  0, "Position Z desire",     1., 0, 100)

exit(gen.generate(PACKAGE, "mur_height_control", "MurHeightControl"))
